index.js

```diff
@@ -2,10 +2,11 @@ var bugsnag = require('bugsnag'),
     util = require('util'),
     winston = require('winston')
 
-function Bugsnag () {
-
-  this.name = 'bugsnag'
+function Bugsnag (options) {
+  options = options || {};
 
+  this.name = 'bugsnag';
+  this.level = options.level || 'silly';
```

Comment:
Winston has a 'level' property to specify the maximum level of messages that should be logged by the transport. I ran into an issue where this wasn't being passed to winston-bugsnag so it'll be great if this change could be merged. :)